CONTRIBUTING.md

```diff
@@ -75,21 +75,19 @@ refer to the [style guide](contributing-guides/style-guide.md).
 
 Translation of pages can be done by simply creating the corresponding page within the appropriate language-specific directory, creating that as well if it does not already exist.
 
-Language specific directories must follow the pattern `pages.<language_tag>`, where `<language_tag>` is a [BCP 47](https://tools.ietf.org/html/bcp47) conforming tag in the form of `<language>[-<region>]`, where:
+Language specific directories must follow the pattern `pages.<locale>`, where `<locale>` is a [POSIX Locale Name](https://www.gnu.org/software/gettext/manual/html_node/Locale-Names.html#Locale-Names) in the form of `<language>_<country>`, where:
```

Comment:
Isn't the `_<country>` optional according to your text below ?

Reply:
@agnivade hmm, yes it is, are you suggesting the addition of square brackets around `_<country>`?

Reply:
Right.

Reply:
Done :+1: